docs/mp/websocket.adoc

```diff
@@ -174,6 +174,47 @@ All endpoint methods in Helidon MP are executed in a separate thread pool, indep
 
 For more information see the link:{helidon-github-tree-url}/examples/webserver/websocket[example].
 
+=== WebSocket Endpoints on Different Ports
+
+The Helidon Reactive Webserver can listen on multiple ports or sockets. This can be useful when APIs for different type of users need to be exposed (such as admin vs. non-admin users). Just like for REST resources, it is possible to expose  WebSocket applications on different ports _provided that the routing paths are different_ --this is due to a constraint in Tyrus, given that it is simply unaware that the endpoints are bound to different ports in the Helidon Webserver.
+
+NOTE: In practice, this implies that the value of `@RoutingPath`, or the equivalent entry in config, must be different across sockets to satisfy the restriction in Tyrus.
```

Comment:
This is not a doc issue, but is there/could there be runtime checking in the code to _make sure_ that the routing paths are different and throw an exception if not?

Reply:
There is an exception being thrown yes, and the message should be clear.

Reply:
Nice. Then maybe the doc could mention that the code does this checking and cite the exception users can expect?

Reply:
Added a sentence about the exception.